### DISCUSSION TITLE
业务高低峰

### DISCUSSION THREAD

**Discussion by woshiyanghai** [hhyo/Archery#2728] [General]
设置一个业务低峰，在这个时间段，业务组长可以审核ddl工单，其他时段，业务组长只能审核dml？这样是否合理

**Comment by LeoQuote** [comment]
#2348 之后, 支持开发者自己覆盖审核引擎, 可继承原引擎并覆盖若干方法, 应该可以达到你的目的

**Reply by woshiyanghai** [reply]
如果在配置项中设置一个时间范围，然后在新增一个函数类似于on_correct_time_period 判断一下？这样ok么？
